src/components/Auth/Registration/Registration.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Box, TextField } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useAppDispatch } from '../../../store/hooks';
+import { signin } from '../../../store/auth/authSlice';
+import RoutesList from '../../../navigation/routes';
+import { SubmitButton, SubmitLink } from '../Auth.style';
+
+function Registration(): JSX.Element {
+  const [t] = useTranslation();
+
+  const dispatch = useAppDispatch();
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  return (
+    <Box component="form" onSubmit={handleSubmit} noValidate>
+      <TextField
+        margin="normal"
+        required
+        fullWidth

```

Comment:
can we make some kind of commonProps and spread them ?

Reply:
fixed